test/controllers/bill_runs.controller.test.js

```diff
@@ -75,7 +78,28 @@ describe('Bill Runs controller', () => {
     Nock.cleanAll()
   })
 
-  describe('Adding a bill run: POST /v2/{regimeSlug}/bill-runs', () => {
+  describe('Creating a bill run: POST /v2/{regimeSlug}/bill-runs', () => {
+    let requestPayload
+    let createdBillRun
+    let createStub
+    let guardStub
+    let response
+
+    beforeEach(async () => {
+      createdBillRun = await NewBillRunHelper.create(authorisedSystem.id, regime.id, { ruleset: 'presroc' })
```

Comment:
From a quick scan of the tests, it looks like we are just asserting the response contains the same bill run ID.

So, I think we could get away with a dummy object and avoid having to make a call to the DB to create a bill run.

```suggestion
      dummyBillRun = { billRun: { id: '050f8351-b796-4bde-ae2f-c524c3bcf94b' } }
```

What do you think?